launch/sim_iarc7_safety.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
iarc7_launch can know about other packages yaml files. It already reaches into different packages to run the different launch scripts based on what yaml files it want's loaded. I think we should change that pattern and instead take an argument to the correct yaml file to load.

In the case of iarc7_safety the rosparams need to know about other packages that are being launched from iarc7_launch so I agree the list should be passed in as a parameter.